.mailmap

```diff
@@ -97,6 +97,7 @@ Lorena Pantano <lorena.pantano@gmail.com> Lorena <lorena.pantano@gmail.com>
 Luis Pedro Coelho <luis@luispedro.org> Luis Pedro Coelho <lpc@cmu.edu>
 Luke Zoltan Kelley <lzkelley@gmail.com> Luke <lzkelley@gmail.com>
 Maarten Breddels <maartenbreddels@gmail.com> maartenbreddels <maartenbreddels@gmail.com>
+Madhumitha Natarajan <galt4000@gmail.com> Madhu94 <galt4000@gmail.com>
 Madhumitha Natarajan <galt4000@gmail.com> nmadhum <nmadhum@deshaw.com>
```

Comment:
Should the original line have been replaced?

Reply:
No, they are taking care of different commits, IIRC.